Products/Zuul/routers/__init__.py

```diff
@@ -161,6 +155,27 @@ def gunzip_b64(self, string):
                     return DirectResponse.exception(e, 'Unable to decompress data')
         return DirectResponse.succeed(data=Zuul.marshal({'data': data}))
 
+    def getGraphLink(self, data):
+        """
+        Try to compress config, if it longer then maxDataLengt save config in
+        Redis
+        """
```

Comment:
This doc-string should be updated with a more accurate description of the method.